app/Http/Controllers/Service/Admin/VouchersController.php

```diff
@@ -47,20 +50,63 @@ public function create()
      */
     public function storeBatch(Request $request)
     {
-        $new_vouchers = [];
+        $newVouchers = [];
```

Comment:
I think we snake_case our laravel php vars and camelCase our js vars usually but obviously not a deal breaker.